tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,32 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite. It uses RRFS forecasts mapped onto 3-km regular grid (rrfs*.conus.grib2) for
+    ICs and LBCs. This test uses old v1 sfc_data, not the v2 fractional grid sfc_data.
```

Comment:
You didn't explain v1 vs. v2 sfc_data and what switches control old vs new behavior.

Reply:
I do not know more details on the differences, as this is the information I was given about the need of new v2 surface data when the full RRFS fields are used (future development), vs. the old v1 surface data required for the interpolated RRFS fields that are being implemented at the moment. The information about v1 and v2 sfc_data targets users who are more familiar with the expanded features of full-grid RRFS data and who could then make more sense of the limitations coming from current use of RRFS data mapped onto a regular 3-km grid.